include/ppx/grfx/vk/vk_device.h

```diff
@@ -90,23 +91,37 @@ class Device
     Result ConfigureQueueInfo(const grfx::DeviceCreateInfo* pCreateInfo, std::vector<float>& queuePriorities, std::vector<VkDeviceQueueCreateInfo>& queueCreateInfos);
     Result ConfigureExtensions(const grfx::DeviceCreateInfo* pCreateInfo);
     Result ConfigureFeatures(const grfx::DeviceCreateInfo* pCreateInfo, VkPhysicalDeviceFeatures& features);
+    void   ConfigureShadingRate(
```

Comment:
nit: ConfigureShadingRate**Capabilities**? and same for the two functions below

Reply:
These have been renamed.